src/helpers/validator.js

```diff
@@ -225,6 +225,20 @@ const isValidRoleName = field => check(field)
   .isIn(['reader', 'author', 'admin', 'superadmin'])
   .withMessage(`${field} can only be one of: 'reader', 'author', 'admin', 'superadmin'`);
 
+/**
+   * @param {String} field
+   * @returns {Object} - Express-validator
+   */
+const isValidReadInput = field => check(field)
+  .exists()
+  .withMessage(`${field} is a required field`)
+  .trim()
+  .not()
+  .isEmpty()
```

Comment:
Nice work, I don't think it is necessary to check if the field is empty when it is enumerable, it can either be true or false

Reply:
Thank you very much for that observation. It makes sense. I'll make the needed changes.

Reply:
LGTM